src/engine/engine.cpp

```diff
@@ -821,6 +821,24 @@ void DivEngine::runExportThread() {
   size_t fadeOutSamples=got.rate*exportFadeOut;
   size_t curFadeOutSample=0;
   bool isFadingOut=false;
+
+  setConf("arcadeCore",getConfInt("arcadeCoreRender",0));
```

Comment:
This looks like a hacky solution. I say extend initDispatch by adding an isRender bool and checking there.